src/app/subnets/views/SubnetDetails/SubnetStaticIPs/SubnetStaticIPs.tsx

```diff
@@ -27,6 +34,7 @@ const SubnetStaticIPs = () => {
           </Button>
         </MainToolbar.Controls>
       </MainToolbar>
+      {IS_DEV && <StaticDHCPTable staticDHCPLeases={mockStaticDHCPLeases} />}
```

Comment:
All the static IP work is already hidden if `VITE_APP_STATIC_IPS_ENABLED` isn't true, so you can remove the condition here and just always display the table

Reply:
Okay, this means we'd need to update the acceptance criteria for this.